### ISSUE TITLE
Question about the detector?

### ISSUE THREAD

**Issue by taylover-pei** [houliang428/CNN-for-face-anti-spoofing#5]
Hello, I have a question about the face detector. What's the file name: "haarcascade_frontalface_default.xml", is it a face detector?

**Comment by houliang428** [comment]
Not really. It is a haar cascade classifier for object detection proposed by Viola and Jones. I used this classifier for detecting faces in images.
